### PR TITLE
Add specific status option to config

### PR DESCRIPTION
**Pull request by tomburns**
Scratching my own itch re bitrise-io/bitrise-steplib#861

This is my first time working on one of these steps, so feedback welcome :)

### CODE REVIEW

**Comment by viktorbenei** [comment]
LGTM, although I'd add a `value_options` to the `set_specific_status`, so that you don't have to manually type that in, just select from a dropdown.

@godrei WDYT?

**Comment by viktorbenei** [comment]
Ohh, and of course, thanks for the PR @tomburns ! ;)

**Comment by tomburns** [comment]
Pushed revisions, thanks! I didn't realize `value_options` was a thing :)

File: step.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
 ci_build_url = ENV['build_url']
 authorization_token = ENV['auth_token']
 build_is_green = ENV['STEPLIB_BUILD_STATUS'] == '0'
+specific_status = ENV['set_specific_status']
```
Comment (godrei) [inline step.rb]: Please print this config also in `puts 'Config:'` section!

Comment (tomburns) [inline step.rb]: @godrei added, thanks for the feedback!

**Comment by tomburns** [comment]
I believe this is ready for review again; Thanks for your time in reviewing!

**Comment by godrei** [comment]
@tomburns ,
awesome! Thanks for the contribution! 👌

**Pull request merged**